### PR TITLE
patch

### PR DESCRIPTION
**Pull request by waynebruce0x**

### CODE REVIEW

**Comment by llamatester** [comment]
The alien-base-amm adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       123 passed, 123 total
    Snapshots:   0 total
    Time:        0.346 s
    Ran all test suites.
    
    Nb of pools: 17
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬────────┬──────────────────┬─────────────────┬────────────────────┬────────────────────┬───────────┬───────────┬──────────────────────────────────────────────────┬──────────────────────────────────────────────────┬──────────────────────────────────────────────────────────────────────────────────────┬─────────────┬─────────────┐
    │ (index) │                     pool                     │ chain  │     project      │     symbol      │       tvlUsd       │      apyBase       │ apyBase7d │ apyReward │                   rewardTokens                   │                 underlyingTokens                 │                                         url                                          │ volumeUsd1d │ volumeUsd7d │
    ├─────────┼──────────────────────────────────────────────┼────────┼──────────────────┼─────────────────┼────────────────────┼────────────────────┼───────────┼───────────┼──────────────────────────────────────────────────┼──────────────────────────────────────────────────┼──────────────────────────────────────────────────────────────────────────────────────┼─────────────┼─────────────┤
    │    0    │ '0x9BB646BF0F4Da44bfaF3d899e774DE065731EDFe' │ 'Base' │ 'alien-base-amm' │  'CBETH-WETH'   │  508385.04217044   │ 5.154744263938089  │     0     │     0     │                        []                        │ [ '0x2Ae3F1Ec7F1F5012CFEab0185bfc7aa3cf0DEc22' ] │ 'https://app.alienbase.xyz/add/0x2Ae3F1Ec7F1F5012CFEab0185bfc7aa3cf0DEc22/undefined' │      0      │      0      │
    │    1    │ '0xd97a40434627D5c897790DE9a3d2E577Cba5F2E0' │ 'Base' │ 'alien-base-amm' │   'DAI+-USD+'   │ 359173.9614094647  │ 7.296171664884345  │     0     │     0     │                        []                        │ [ '0x65a2508C429a6078a7BC2f7dF81aB575BD9D9275' ] │ 'https://app.alienbase.xyz/add/0x65a2508C429a6078a7BC2f7dF81aB575BD9D9275/undefined' │      0      │      0      │
    │    2    │ '0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4' │ 'Base' │ 'alien-base-amm' │      'ALB'      │ 234770.76791416487 │ 195.3412292656773  │     0     │     0     │ [ '0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4' ] │ [ '0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4' ] │                           'https://app.alienbase.xyz/swap'                           │      0      │      0      │
    │    3    │ '0x553666081db0a8fdB337560009932852059d589A' │ 'Base' │ 'alien-base-amm' │  'USD+-USDBC'   │ 184263.6869056229  │ 14.221982225625547 │     0     │     0     │                        []                        │ [ '0xB79DD08EA68A908A97220C76d19A6aA9cBDE4376' ] │ 'https://app.alienbase.xyz/add/0xB79DD08EA68A908A97220C76d19A6aA9cBDE4376/undefined' │      0      │      0      │
    │    4    │ '0xAcBFF2Edbcf8089AdeC9706FF3cB6DF2502A30E1' │ 'Base' │ 'alien-base-amm' │ 'USDBC-AXLUSDC' │ 183319.0422671445  │ 14.295268225223964 │     0     │     0     │                        []                        │ [ '0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA' ] │ 'https://app.alienbase.xyz/add/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/undefined' │      0      │      0      │
    │    5    │ '0xeF24722d5daE32Dc155d961561cFFbc5f347EeE7' │ 'Base' │ 'alien-base-amm' │  'WETH-USDBC'   │ 141635.14821586089 │ 37.00486656046772  │     0     │     0     │                        []                        │ [ '0x4200000000000000000000000000000000000006' ] │ 'https://app.alienbase.xyz/add/0x4200000000000000000000000000000000000006/undefined' │      0      │      0      │
    │    6    │ '0xd79Ef386f26E40B5e60e4aeB324F2678F6eBbF49' │ 'Base' │ 'alien-base-amm' │   'ALB-WETH'    │ 136580.30402379582 │ 335.7761628060949  │     0     │     0     │                        []                        │ [ '0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4' ] │ 'https://app.alienbase.xyz/add/0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4/undefined' │      0      │      0      │
    │    7    │ '0xB4A9960b23D056E8B42c6EDBEe7407cF92695223' │ 'Base' │ 'alien-base-amm' │   'DAI-USDBC'   │ 136553.1100229053  │ 9.595515179260374  │     0     │     0     │                        []                        │ [ '0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb' ] │ 'https://app.alienbase.xyz/add/0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb/undefined' │      0      │      0      │
    │    8    │ '0x6C15F130217143Fd3f870D4451457C617ac9A632' │ 'Base' │ 'alien-base-amm' │ 'REBASE-USDBC'  │ 103868.71393182843 │         0          │     0     │     0     │                        []                        │ [ '0x3421cc14F0e3822Cf3B73C3a4BEC2A1023b8d9Cf' ] │ 'https://app.alienbase.xyz/add/0x3421cc14F0e3822Cf3B73C3a4BEC2A1023b8d9Cf/undefined' │      0      │      0      │
    │    9    │ '0xe9B0A6bc01B9b4eE44C7B012EC7d5A5EEAcc40df' │ 'Base' │ 'alien-base-amm' │   'ALB-USDBC'   │ 59248.04588854868  │ 387.0204469381816  │     0     │     0     │                        []                        │ [ '0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4' ] │ 'https://app.alienbase.xyz/add/0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4/undefined' │      0      │      0      │
    └─────────┴──────────────────────────────────────────────┴────────┴──────────────────┴─────────────────┴────────────────────┴────────────────────┴───────────┴───────────┴──────────────────────────────────────────────────┴──────────────────────────────────────────────────┴──────────────────────────────────────────────────────────────────────────────────────┴─────────────┴─────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

File: src/adaptors/alien-base-amm/index.js [hunk 1]
```diff
@@ -17,7 +17,13 @@ const getPoolDetails = async (block, poolInfo, chainString) => {
   // console.log(poolInfo);
   for (let i = 0; i < poolInfo.length; i++) {
     // SKIP LP OF ALB STANDALONE
-    if (poolInfo[i].lpToken != '0x1dd2d631c92b1aCdFCDd51A0F7145A50130050C4') {
```
Comment (christianlantin) [inline src/adaptors/alien-base-amm/index.js:20]: Master

**Review by christianlantin — COMMENTED** [review]
Master

File: src/adaptors/alien-base-amm/index.js (hunk 1 above)
Comment (christianlantin) [reply src/adaptors/alien-base-amm/index.js:20]: Master

**Pull request merged**